src/util.h

```diff
@@ -4,6 +4,15 @@
 #ifndef _UTIL_H
 #define _UTIL_H
 
+#if __SANITIZE_ADDRESS__
+#define P11PROV_ADDRESS_SANITIZER 1
+#endif
+#if defined(__has_feature)
+#if __has_feature(address_sanitizer)
+#define P11PROV_ADDRESS_SANITIZER 1
+#endif
+#endif
+
```

Comment:
This sounds like it should go in provider.h ?

Reply:
moved there for now